aptos-move/aptos-gas/src/misc.rs

```diff
@@ -13,14 +13,14 @@ crate::params::define_gas_parameters!(
     "misc.abs_val",
     [
         // abstract value size
-        [u8: AbstractValueSize, "u8", 1],
-        [u64: AbstractValueSize, "u64", 8],
-        [u128: AbstractValueSize, "u128", 16],
-        [bool: AbstractValueSize, "bool", 1],
-        [address: AbstractValueSize, "address", 32],
-        [struct_: AbstractValueSize, "struct", 8],
-        [vector: AbstractValueSize, "vector", 16],
-        [reference: AbstractValueSize, "reference", 16],
+        [u8: AbstractValueSize, "u8", 40],
```

Comment:
Bytesize is [48 for ValueImpl](https://github.com/move-language/move/issues/453#issuecomment-1266243638). In any case, I'm not sure whether we should charge here for every primitive the same just because we have a sub-optimal implementation. It incentivizes people to use the highest bit precision (u256 eventually) because all cost the same. I would use the conceptual memory use here, not the one in out current impl.

@movekevin @davidiw